.changeset/odd-cats-turn.md

```diff
@@ -0,0 +1,7 @@
+---
+"wrangler": patch
+---
+
+Check for assets in config file as well as CLI arg
```

Comment:
This changeset doesn't make it clear why the check is there (to disable the mandatory requirement for an entry point)

Reply:
You didn't address this? :(

Reply:
Fine, I'll fix it myself.